types/promise-retry/index.d.ts

```diff
@@ -24,4 +24,6 @@ type RetryableFn<ResolutionType> = ((retry: (error: any) => never, attempt: numb
  */
 declare function promiseRetry<ResolutionType>(retryableFn: RetryableFn<ResolutionType>, options?: OperationOptions): Promise<ResolutionType>;
 declare function promiseRetry<ResolutionType>(options: OperationOptions, retryableFn: RetryableFn<ResolutionType>): Promise<ResolutionType>;
+// Exporting a namespace too
+declare namespace promiseRetry {}
```

Comment:
Could you provide the use case for this (playground link or, preferably, tests)? Consumers of this library should use the `require` syntax for importing or have the `esModuleInterop` flag + `import x from 'promise-retry'` syntax. This namespace change shouldn't be needed if the module is consumed the right way.

Reply:
I believe there are legitimate reasons that consumers of this package don't want to import using commonjs `require` nor enable the global `esModuleInterop` configuration option, just to consume this package. For one thing, this package claims to support TypeScript 2.3, the `esModuleInterop` was first [introduced with TypeScript 2.7](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html) and the `esModuleInterop` flag is not enabled by default.